docs/technical_reference/unit_models/zero_order_unit_models/mabr_zo.rst

```diff
@@ -16,6 +16,26 @@ Costing Method
 Costing is calculated using the cost_mabr method in the zero-order costing package.
 See documentation for the :ref:`zero-order costing package<zero_order_costing>`.
 
+Additional Variables
+--------------------
+
+.. csv-table::
+   :header: "Description", "Variable Name", "Units"
+
+   "Nitrogen removal rate per day", "nitrogen_removal_rate", "g/d/m**2"
+   "Sizing variable for effective reactor area", "reactor_area", "m**2"
```

Comment:
```suggestion
   "Nitrogen removal rate per day", "nitrogen_removal_rate", "g/d/m\ :sup:`2`"
   "Sizing variable for effective reactor area", "reactor_area", "m\ :sup:`2`"
```

Reply:
I think I'll pass on these superscript mods for now since I am modifying everything through a single automation script, and this would require more effort than I think would be worthwhile at this point. I say this mostly because the bold rendering that GH shows, does _not_ occur in read the docs, so it is technically not an issue. However, it would be a nice-to-have to include actual superscripts rather than the `**` operator.